tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml

```diff
@@ -59,31 +62,36 @@ jobs:
     parameters:
       packageFolder: '$(Build.BinariesDirectory)\nuget-artifact'
 
+  # TODO: Add .Net Core AnyCPU test task
+
   - script: |
      @echo "Running Runtest.bat"
      test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact netcoreapp2.1 x64 $(NuGetPackageVersionNumber)
     workingDirectory: '$(Build.SourcesDirectory)\csharp'
-    displayName: 'Run End to End Test (C#) .Net Core'
+    displayName: 'Run End to End Test (C#) .Net Core x64'
 
-  - script: |
-     @echo "Running Runtest.bat"
-     test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact netcoreapp2.1 x86 $(NuGetPackageVersionNumber)
-    workingDirectory: '$(Build.SourcesDirectory)\csharp'
-    displayName: 'Run End to End Test (C#) .Net Core x86'
-    enabled: false
+  - ${{ if ne(parameters['Skipx86Tests'], 'true') }}:
+      - script: |
+         @echo "Running Runtest.bat"
+         test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact netcoreapp2.1 x86 $(NuGetPackageVersionNumber)
+        workingDirectory: '$(Build.SourcesDirectory)\csharp'
+        displayName: 'Run End to End Test (C#) .Net Core x86'
 
-  - script: |
-     @echo "Running Runtest.bat"
-     test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact net461 x64 $(NuGetPackageVersionNumber)
-    workingDirectory: '$(Build.SourcesDirectory)\csharp'
-    displayName: 'Run End to End Test (C#) .NetFramework'
+  # TODO: Add .Net Framework AnyCPU test task
 
   - script: |
      @echo "Running Runtest.bat"
-     test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact net461 x86 $(NuGetPackageVersionNumber)
+     test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact net461 x64 $(NuGetPackageVersionNumber)
     workingDirectory: '$(Build.SourcesDirectory)\csharp'
-    displayName: 'Run End to End Test (C#) .NetFramework x86'
-    enabled: false
+    displayName: 'Run End to End Test (C#) .NetFramework x64'
+
+  - ${{ if ne(parameters['Skipx86Tests'], 'true') }}:
+      - script: |
+         @echo "Running Runtest.bat"
+         test\Microsoft.ML.OnnxRuntime.EndToEndTests\runtest.bat $(Build.BinariesDirectory)\nuget-artifact net461 x86 $(NuGetPackageVersionNumber)
+        workingDirectory: '$(Build.SourcesDirectory)\csharp'
+        displayName: 'Run End to End Test (C#) .NetFramework x86'
+        enabled: false
```

Comment:
Keep .Net framework x86 tests disabled because it is currently failing in CI (but passes locally).  Will track enabling this as a separate wok item.